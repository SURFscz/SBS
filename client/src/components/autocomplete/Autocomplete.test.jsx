import React from "react";
import { render } from "@testing-library/react";
import AutoComplete from "./Autocomplete";

jest.mock("../../locale/I18n", () => ({
  t: key => key
}));

const options = [
  { label: "Option 1", value: "option1" },
  { label: "Option 2", value: "option2" }
];

describe("AutoComplete", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(
      <AutoComplete
        options={options}
        value={options[0]}
        onChange={jest.fn()}
        placeholder="Select an option"
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
