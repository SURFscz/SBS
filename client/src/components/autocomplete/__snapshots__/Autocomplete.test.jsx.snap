// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutoComplete matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="autocomplete"
  />
</DocumentFragment>
`;
